client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -1006,6 +1034,91 @@ export default function CampaignItemDetails( props: Props ) {
 									) }
+									{ tsp && (
+										<>
+											<div className="campaign-item-details__main-stats-row ">
+												<div className="campaign-item-details__main-stats-title">
+													<span className="campaign-item-details__title">
+														{ translate( 'Social Engagement' ) }
+													</span>
+													<a
+														href={ permalink }
+														target="_blank"
+														rel="noreferrer"
+														className="campaign-item-details__tsp-permalink"
+													>
+														<span>{ translate( 'Open ad preview' ) }</span>
+														<Gridicon icon="external" size={ 16 } />
+													</a>
+												</div>
+												<div>
+													<span className="campaign-item-details__label">
+														{ translate( 'Tumblr Post views' ) }
+													</span>
+													<span className="campaign-item-details__text">
+														<span className="wp-brand-font">
+															{ ! isLoading ? tspImpressionsFormatted : <FlexibleSkeleton /> }
+														</span>
+													</span>
+												</div>
+												{ /* commenting this until we figure out if this is working properly*/ }
+												{ /*<div>*/ }
+												{ /*	<span className="campaign-item-details__label">*/ }
+												{ /*		{ translate( 'Site visits from Tumblr Post' ) }*/ }
+												{ /*	</span>*/ }
+												{ /*	<span className="campaign-item-details__text">*/ }
+												{ /*		<span className="wp-brand-font">*/ }
+												{ /*			{ ! isLoading ? tspClicksFormatted : <FlexibleSkeleton /> }*/ }
+												{ /*		</span>*/ }
+												{ /*	</span>*/ }
+												{ /*</div>*/ }
+											</div>
+											<div className="campaign-item-details__main-stats-row ">
+												<div>
+													<span className="campaign-item-details__label">
+														{ translate( 'Replies' ) }
+													</span>
+													<span className="campaign-item-details__text">
+														<span className="wp-brand-font">
+															{ ! isLoading ? repliesFormatted : <FlexibleSkeleton /> }
+														</span>
+													</span>
+												</div>
+												<div>
+													<span className="campaign-item-details__label">
+														{ translate( 'Likes' ) }
+													</span>
+													<span className="campaign-item-details__text">
+														<span className="wp-brand-font">
+															{ ! isLoading ? likesFormatted : <FlexibleSkeleton /> }
+														</span>
+													</span>
+												</div>
+												{ displayedReplies && replies && replies?.total_notes > 0 && (
+													<div className="campaign-items-details__tsp-replies">
+														{ displayedReplies.map( ( note, index ) => (
+															<div key={ index } className="campaign-items-details__tsp-reply">
+																<a href={ note.blog_url } target="_blank" rel="noopener noreferrer">
+																	@{ note.blog_name }
+																</a>
+																<br />
```

Comment:
We should probably add something like the following to output actual reply test:
```js
{ note?.reply_text && note.reply_text }
```

Reply:
added.. thanks